b2b-sample/README.md

```diff
@@ -65,6 +67,9 @@ Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attri
 >| Authorized redirect URLs | `http://localhost:3001/api/auth/callback/wso2isAdmin` and `http://localhost:3001` |
 >| Allowed origin           |                           `http://localhost:3001`                            |
 
+Also, On the User Attributes tab, click on + Add User Attributes.
+Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attributes.
+
```

Comment:
Moved from bottom as mentioned in this comment https://github.com/wso2/samples-is/pull/405/files#r1133117678